Corona-Warn-App/src/test/java/de/rki/coronawarnapp/datadonation/analytics/modules/registeredtest/TestResultDonorTest.kt

```diff
@@ -146,8 +144,7 @@ class TestResultDonorTest : BaseTest() {
             every { testResultDonorSettings.testResultAtRegistration } returns mockFlowPreference(CoronaTestResult.PCR_POSITIVE)
             every { testResultDonorSettings.finalTestResultReceivedAt } returns mockFlowPreference(baseTime)
 
-            val donation = testResultDonor.beginDonation(TestRequest)
-            donation.shouldBeInstanceOf<TestResultDonor.TestResultMetadataContribution>()
+            val donation = testResultDonor.beginDonation(TestRequest) as TestResultDonor.TestResultMetadataContribution
```

Comment:
To avoid
```java
java.lang.reflect.MalformedParameterizedTypeException
	at java.base/sun.reflect.generics.reflectiveObjects.ParameterizedTypeImpl.validateConstructorArguments(ParameterizedTypeImpl.java:59)
	at java.base/sun.reflect.generics.reflectiveObjects.ParameterizedTypeImpl.<init>(ParameterizedTypeImpl.java:52)
	at java.base/sun.reflect.generics.reflectiveObjects.ParameterizedTypeImpl.make(ParameterizedTypeImpl.java:93)
	at java.base/sun.reflect.generics.factory.CoreReflectionFactory.makeParameterizedType(CoreReflectionFactory.java:105)
	at java.base/sun.reflect.generics.visitor.Reifier.visitClassTypeSignature(Reifier.java:140)
	at java.base/sun.reflect.generics.tree.ClassTypeSignature.accept(ClassTypeSignature.java:49)
	at 
        java.base/sun.reflect.generics.repository.ClassRepository.computeSuperInterfaces(ClassRepository.java:117)
	at java.base/sun.reflect.generics.repository.ClassRepository.getSuperInterfaces(ClassRepository.java:95)
	at java.base/java.lang.Class.getGenericInterfaces(Class.java:1137)
	at kotlin.jvm.internal.ReflectionFactory.renderLambdaToString(ReflectionFactory.java:44)
```